sdk/typescript/libraries/botbuilder-solutions/src/authentication/multiProviderAuthDialog.ts

```diff
@@ -166,7 +166,6 @@ export class MultiProviderAuthDialog extends ComponentDialog {
 
     private async sendRemoteEvent(stepContext: WaterfallStepContext): Promise<DialogTurnResult> {
         if (isRemoteUserTokenProvider(stepContext.context.adapter)) {
-            // eslint-disable-next-line @typescript-eslint/tslint/config, @typescript-eslint/no-explicit-any
             const tokenProvider: IRemoteUserTokenProvider = <any>stepContext.context.adapter;
```

Comment:
I think it's perfect to remove the comment disabling the rule, but it shouldn't be left with the `<any>` casting, does this change work to fix the issue?
```suggestion
             const tokenProvider: IRemoteUserTokenProvider = <IRemoteUserTokenProvider>stepContext.context.adapter;
```